lib/src/component2_suggestors/component2_constants.dart

```diff
@@ -0,0 +1,17 @@
+const revertInstructions = '/// These updates are opt-in and can be '
+    'rolled back if you do not wish to do the update at this time.';
+
+String methodDeprecationMessage({methodName}) =>
+    '/// FIXME: [$methodName] has been '
+    'deprecated and should be updated to [unsafe_$methodName]. Please see '
```

Comment:
Should our comment also link them to instructions _(something we should eventually add as an .md file within over_react - or maybe the wiki)_ for how to get off of `unsafe_` methods completely and use the new React16 lifecycle method replacements?  

I'm thinking many teams - especially top-level experiences - will be extremely wary of committing code that involves the use of a method with a name that starts with the word **unsafe** 😉

Reply:
That's a really good idea. I added the link to the React docs because that seemed to be the best source at the moment, but let me know if there's a different doc that's better!

I also updated the comment wording because I realized we don't expose the `unsafe` version of the lifecycle method in `Component2`, we just throw an unsafe error. So I reworded that to be more clear, along with adding the link!

Reply:
Sounds good.